homework/index.js

```diff
@@ -1,19 +1,13 @@
 'use strict';
 
 {
-  function fetchJSON(url, cb) {
-    const xhr = new XMLHttpRequest();
-    xhr.open('GET', url);
-    xhr.responseType = 'json';
-    xhr.onload = () => {
-      if (xhr.status >= 200 && xhr.status <= 299) {
-        cb(null, xhr.response);
-      } else {
-        cb(new Error(`Network error: ${xhr.status} - ${xhr.statusText}`));
+  function fetchJSON(url) {
+    return fetch(url).then(response => {
+      if (!response.ok) {
+        throw new Error(`Error occured ${response.status} ${response.statusText}`);
```

Comment:
Make sure to correct spelling errors flagged by VSCode: `occured` -> `occurred`.

Reply:
Fixed typo. Thanks for the tip